EXILED/Exiled.API/Features/Player.cs

```diff
@@ -687,24 +687,24 @@ public ScpSpawnPreferences.SpawnPreferences ScpPreferences
         /// Equivalent to checking the player's <see cref="Team"/>.
         /// </summary>
         // TODO: Change logic for FacilityGuard in next major update
-        public bool IsNTF => Role?.Team is Team.FoundationForces;
+        public bool IsNTF => Role?.Type.IsNtf() ?? false;
 
         /// <summary>
         /// Gets a value indicating whether or not the player's <see cref="RoleTypeId"/> is any Chaos rank.
         /// Equivalent to checking the player's <see cref="Team"/>.
         /// </summary>
-        public bool IsCHI => Role?.Team is Team.ChaosInsurgency;
+        public bool IsCHI => Role?.Type.IsChaos() ?? false;
 
         /// <summary>
         /// Gets a value indicating whether or not the player's <see cref="RoleTypeId"/> is any SCP.
         /// Equivalent to checking the player's <see cref="Team"/>.
         /// </summary>
-        public bool IsScp => Role?.Team is Team.SCPs;
+        public bool IsScp => Role?.Type.IsScp() ?? false;
 
         /// <summary>
         /// Gets a value indicating whether or not the player's <see cref="RoleTypeId"/> is any human rank.
         /// </summary>
-        public bool IsHuman => Role is not null && Role.Is(out HumanRole _);
+        public bool IsHuman => Role is not null && RoleExtensions.IsHuman(Role.Type);
```

Comment:
Just call it on the RoleType itself; `Role.Type.IsHuman()`

Reply:
Doesn't work as the base game has IsHuman()

Reply:
> Doesn't work as the base game has IsHuman()

if it's already exist we shouldn't make an other same extension because it's would create conflict

Reply:
This ^

Reply:
Also shouldnt need to modify it at all checking Role is HumanRole works and should never be incorrect